MystenLabs/sui
# [example, not for commit] deepbook indexer example

## Description 
The best way to start in my opinion is to look at `deepbook_worker.rs` and start writing the deepbook logic. 
Everything else will come from there.
I am leaving few comments in the PR, not code, to highlight something.
In any case we talked about it and let's talk more

## Test plan 

Local run with local postgres setup and diesel commands to change the schema (which you'll have to do a lot).
- install postgres, which you may have already
- install diesel - https://diesel.rs/guides/getting-started.html which should give you enough info to also start the DB.
Then you can run and see transactions coming and you can play with them and debug things.
With proper checkpoint ranges you can have a very fast dev cycle, changing schema and all

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates.
